readme.md

```diff
@@ -1,69 +1,58 @@
-> Please help us improve and share your feedback! If you find better tutorials
-> or links, please share them by [opening a pull request](https://github.com/HackYourFuture-CPH/React/pulls).
-
 <h1 align='center'>
  HackYourFuture - React
-</h1>   
+</h1>
+
+> [!NOTE]
+> Please help us improve and share your feedback! If you find better tutorials or links, please share them by [opening a pull request](https://github.com/HackYourFuture-CPH/React/pulls).
 
 <p align='center'>
-<img src='react-logo.png' width='600px'>  
+<img src='assets/react-logo.png' width='600px'>
 </p>   
 
 
-React is free an open-source frontend JavaScript library for building user interfaces based on UI components.
-
-This course is divided in 2 modules, of 3 and 2+1 week.
-
-
-## Module goals:
-
-### [React1 Module:](/react1/readme.md)   
-- Thinking the React way
-- Building components
-- Advanced React
+React is free an open-source frontend JavaScript library for building user interfaces based on UI components. We will learn React to empower us to build even more complex applications faster.
 
-### [React2 Module:](/react2/readme.md)
-- Managing state
-- Connecting to node
 
-\+ [Individual Project (Meal Sharing App)](https://github.com/HackYourFuture-CPH/React/blob/main/react2/readme.md#individual-project)
+<p style="margin-top: 50px" align='center'>
+<img src='assets/nextjs-logo.png' width='400px'>  
+</p>
 
+Next.js is a popular framework which further enhances the development process and performance of our web applications. Next.js builds on top of React by adding several features and optimizations that are often required in modern web applications. 
 
-## Read up before we get started
-
-Watch this video from Facebook about why they started React:
-
-https://youtu.be/nYkdrAPrdcw
+> [!IMPORTANT]
+> The HackYourFuture curriculum is subject to CC BY copyright. This means you can freely use our materials, but just make sure to give us credit for it :blush:
 
-Also be sure you're familiar with ES6-syntax. This is a handy reference of all features of ES6:
+## Module goals
 
-http://es6-features.org
+This course is divided in 2 modules
 
-Please _do not worry about understanding all_. However, do make sure you understand the following, as we'll be using them a lot:
+### [React 1](/react1/readme.md) (3 weeks)
+Here we will focus on:
+ - Why we learn React
+ - How to write React components in **JSX**
+ - How to use **props** and **state** in components
+ - How to react to events using **hooks**
 
-- http://es6-features.org/#Constants
-- http://es6-features.org/#ObjectMatchingShorthandNotation
-- http://es6-features.org/#RestParameter
-- http://es6-features.org/#SpreadOperator
+### [React 2](/react2/readme.md) (2 weeks)
```

Comment:
"2 weeks" here conflicts a little with the link it goes to.

I suggest we change it here to 3 weeks and include the meal sharing bit in this section too, like:

```
### [React 2](/react2/readme.md) (3 weeks)
...
In the third week, you'll get to implement the UI for your Meal Sharing app...
```